pep-0394.txt

```diff
@@ -5,104 +5,181 @@ Last-Modified: $Date$
 Author: Kerrick Staley <mail@kerrickstaley.com>,
         Nick Coghlan <ncoghlan@gmail.com>,
         Barry Warsaw <barry@python.org>,
-        Petr Viktorin <encukou@gmail.com>
+        Petr Viktorin <encukou@gmail.com>,
+        Miro Hrončok <miro@hroncok.cz>,
 Status: Active
 Type: Informational
 Content-Type: text/x-rst
 Created: 02-Mar-2011
-Post-History: 04-Mar-2011, 20-Jul-2011, 16-Feb-2012, 30-Sep-2014, 28-Apr-2018
+Post-History: 04-Mar-2011, 20-Jul-2011, 16-Feb-2012, 30-Sep-2014, 28-Apr-2018,
+              12-Apr-2019
 Resolution: https://mail.python.org/pipermail/python-dev/2012-February/116594.html
 
 
 Abstract
 ========
 
 This PEP provides a convention to ensure that Python scripts can continue to
-be portable across ``*nix`` systems, regardless of the default version of the
-Python interpreter (i.e. the version invoked by the ``python`` command).
+be portable across ``*nix`` systems, regardless of the version of the Python
```

Comment:
As a point of clarification, are we considering MacOS as a ``*nix`` system?

Reply:
Pedantically, it's "macOS" but I hope the answer is "yes".

Reply:
Let me know if you want e to explicitly put this information in.